app/controllers/admin/alert_types/school_alert_type_exceptions_controller.rb

```diff
@@ -0,0 +1,33 @@
+module Admin
+  module AlertTypes
+    class SchoolAlertTypeExceptionsController < AdminController
+      load_and_authorize_resource :school_alert_type_exception
+      load_and_authorize_resource :alert_type
+
+      def index
+        @exceptions = SchoolAlertTypeException.where(alert_type: @alert_type)
+      end
+
+      def new_multiple
+        @schools = School.all
+      end
+
+      def create_multiple
+        school_ids = params[:school_ids]
+        reasons = params[:school_reasons]
+
```

Comment:
I delete them all, for the alert type and re-create them from what comes back in the form...